test/winrt_calendar_test.dart

```diff
@@ -329,15 +334,6 @@ void main() {
       expect(resolvedLanguage.length, equals(5));
     });
 
-    test('Calendar.numeralSystem getter', () {
```

Comment:
Curious -- is there a reason why this test is not good?

Reply:
This was a duplicate of the test in Line 63.